features/homepage.feature

```diff
@@ -1,29 +1,31 @@
-Feature: Homepage
-  As a user of NICE org 
-  I want to be able to use the NICE homepage
+# Feature: Homepage
```

Comment:
A lot of these tests have been commented out is that intentional?

Reply:
Good question, probably not, will reinstate and update